### PR TITLE
Styling for Create Event, Create Forum Pages

### PR DESCRIPTION
**Pull request by safchow**
## Trello ticket link

[Developer Task: Implement UI/UX feedback to the forms](https://trello.com/c/EVvyVCIw/125-developer-task-implement-ui-ux-feedback-to-the-forms )

## Implementation description
* Add Styling for the create event page
* Add Styling for the create forum page

## Steps to test
1. Verify an event can be created
2. Verify error messages appear when required text fields are not given
3. Verify a forum can be created
4. Verify error messages appear when required text fields are not given

### CODE REVIEW

**Comment by Eshah7** [comment]
Screenshot 2023-03-21 at 11 30 44 PM

Having trouble leaving this comment!

**Review by Eshah7 — COMMENTED** [review]
Screenshot 2023-03-21 at 11 33 31 PM

I think you forgot to change the label inside? :0

**Review by Eshah7 — CHANGES_REQUESTED** [review]
LGTM! its so pretty!!!! let's go CHAKRA UI!! (go india 🇮🇳 )

just the minor edit regarding the labels when creating a forum!

**Comment by safchow** [comment]
> Screenshot 2023-03-21 at 11 33 31 PM
> 
> I think you forgot to change the label inside? :0

true copy-pasting goes so hard until it doesnt

**Review by Eshah7 — APPROVED** [review]
LGTM! 💪

File: src/components/forums/CreateForum.js [hunk 1]
```diff
@@ -207,52 +183,71 @@ const CreateForum = () => {
...
-            </FormControl>
-
-            <Grid item>
-                <SubmitButton
-                    label={"SUBMIT"}
-                    onButtonClick={validateForum}
-                />
-            </Grid>
+            <Card style={{ padding: '16px' }}>
+                <Text align="center" className="form-header">Create an Forum</Text>
```
Comment (gabbytang) [inline src/components/forums/CreateForum.js]: Create "a" forum. Otherwise, looks great!

image

**Review by gabbytang — CHANGES_REQUESTED** [review]

File: src/components/forums/CreateForum.js (hunk 1 above)
Comment (Eshah7) [reply src/components/forums/CreateForum.js]: my esl era got me there oop

Comment (safchow) [reply src/components/forums/CreateForum.js]: another copy paste error 💀

**Review by gabbytang — APPROVED** [review]
Looks great!

**Pull request merged**